src/main/java/com/epam/izh/rd/online/entity/SchoolBook.java

```diff
@@ -20,5 +20,72 @@
  * 6) Переопределить метод toString с выводом всех полей (не забывайте alt+inset)
  */
 public class SchoolBook extends Book {
+    private String authorName;
```

Comment:
why you didn't use 'private Author author;'? what will you do if the author's name would be changed?